R/Connection.R

```diff
@@ -608,24 +608,94 @@ setMethod(
 
 #' List Available ODBC Drivers
 #'
-#' List the available drivers on your system. See the
-#' [Installation](https://github.com/r-dbi/odbc#installation) section of the
-#' package README for details on how to install drivers for the most common
-#' databases.
-#'
-#' @param keep A character vector of driver names to keep in the results, if
-#'   `NULL` (the default) will keep all drivers.
-#' @param filter A character vector of driver names to filter from the results, if
-#'   `NULL` (the default) will not filter any drivers.
+#' Collect information about the available driver names.
```

Comment:
I'd add something like: "A driver must be both installed and recorded in `odbcinst.ini` to be included in this list". Or maybe: "This lists drivers that have been recorded in `odbcinst.ini`; it is always also possible to provide a path to a driver directly".

Reply:
This is a great idea---exactly the concepts that ought to appear in a 3-sentence summary. I _did_ replace the reference to `odbcinst.ini` with a more vague "driver manager" since the `.ini` files are Unix only.

[c99a80b](https://github.com/r-dbi/odbc/pull/649/commits/c99a80b2dd361ccecc5bbce11c63be7fc5f3e514).